stable/fluent-bit/values.yaml

```diff
@@ -151,6 +151,7 @@ fullConfigMap: false
 ##
 existingConfigMap: ""
 
+extraLuaScripts: {}
```

Comment:
This new value is missing in the README.

Disregarding that though, how would one use this value? I guess what's the benefit over just using `extraVolumeMounts`?

Reply:
fixed.

imo, the extraLuaScripts value is a quite simple way to inject additional Lua scripts to the container while existing extraVolumeMounts is not. the main "problem" here is the need to place the scripts content by some way into the volume where extraLuaScripts is a simple plain-text setting.